Documentation/src/chunk.md

```diff
@@ -0,0 +1,30 @@
+# `IEnumerable` extension method `.Chunk`
+With the `.Chunk(int)` extension method, you can turn an `IEnumerable<T>` into a `IEnumerable<IEnumerable<T>>`, with the inner Enumerables being of the given size.
+Empty and negative chunk sizes are not allowed and will throw a `ArgumentOutOfRangeException`.

```

Comment:
Maybe explicitly say that the last inner Sequence might be smaller than size.

Reply:
Covered by 2nd example